js/ui/calendar/ui.calendar.js

```diff
@@ -443,9 +520,9 @@ const Calendar = Editor.inherit({
 
         const date = this._getDateByOffset(offset * this._getRtlCorrection());
 
-        this._moveToClosestAvailableDate(date, offset);
+        this._moveToClosestAvailableDate(date);
 
-        setTimeout((function() {
+        this._waitRenderViewTimeout = setTimeout((function() {
```

Comment:
```suggestion
        this._waitRenderViewTimeout = setTimeout(() => {
```

And remove `).bind(this)` below

Reply:
fixed it